test/flows.spec.js

```diff
@@ -104,8 +102,9 @@ describe('Flows', function() {
         stripe.customers.create(CUSTOMER_DETAILS).then((customer) => {
           cleanup.deleteCustomer(customer.id);
 
-          return stripe.customers
-            .updateSubscription(customer.id, {
+          return stripe.subscriptions
```

Comment:
Some of those tests should likely just be removed right? You're basically just re-testing subscriptions here (though we stopped adding anything to this flow test suite a while ago)

Reply:
+1

Reply:
I wanted to keep the scope for this PR contained to just removing deprecated methods. But yes, we should get rid of `flows.spec.js` entirely and ideally move the rest of the test suite to use stripe-mock instead of local mocks.

Reply:
Can you create a follow-on jira?